deploy/role.yaml

```diff
@@ -71,6 +88,7 @@ rules:
       - mellanox.com
     resources:
       - '*'
+      - macvlannetworks
```

Comment:
`*` means all resources so i dont think you need to add explicitly `macvlannetworks`